frontend/src/components/StudentProfile/GraduationStatus/CertificationDogwoods.vue

```diff
@@ -3,7 +3,7 @@
     <v-card title="Student Certificates/Dogwoods" no-body>
       <v-card-text class="py-4">
         <div
-          v-if="this.studentGradStatus.schoolAtGrad && !isCertificateEligible()"
+          v-if="!isCertificateEligible(studentGradStatus.schoolOfRecordId)"
```

Comment:
We should be using `schoolAtGradID` here instead of `schoolOfRecordId` since the warning message is referring to the student's school at graduation. You'll also have to make sure the student has a schoolAtGraduationId and this can either be handled inline here in the v-if OR in the isCertificateEligible() function.